image/templates/helm/stackrox-secured-cluster/templates/_admission-controller-config.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "srox.protectAdmissionControllerConfig" -}}
+{{- $ := . -}}
+
+{{- $formatMsg := "It is not supported anymore to specify 'admissionControl.%s'. This setting will be ignored." -}}
```

Comment:
But are the settings really ignored?

My understanding is that if the user is manually setting the values, then this will just print the warning messages, but then `applyDefaults` does not override them, so I think the user specified values are still preserved.